### PR TITLE
Added console Links

### PR DESCRIPTION
**Pull request by ajinkya1986**

### CODE REVIEW

File: cartography/intel/azure/__init__.py [hunk 1]
```diff
@@ -218,7 +218,7 @@ def start_azure_ingestion(
     )
 
     if config.azure_sync_all_subscriptions:
-        subscriptions = subscription.get_all_azure_subscriptions(credentials)
+        subscriptions = subscription.get_all_azure_subscriptions(credentials, common_job_parameters)
 
     else:
         subscriptions = subscription.get_current_azure_subscription(
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/__init__.py:224]: add console link in this logic also

File: cartography/intel/azure/cosmosdb.py [hunk 2]
```diff
@@ -650,6 +661,7 @@ def _load_sql_databases(neo4j_session: neo4j.Session, sql_databases: List[Dict],
     sdb.region = database.location
     SET sdb.name = database.name,
     sdb.throughput = database.options.throughput,
+    sdb.consoleLink = database.consolelink,
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/cosmosdb.py:664]: Property name should be `consolelink` from cartography side

File: cartography/intel/azure/cosmosdb.py [hunk 3]
```diff
@@ -679,6 +691,7 @@ def _load_cassandra_keyspaces(neo4j_session: neo4j.Session, cassandra_keyspaces:
     ck.region = keyspace.location
     SET ck.name = keyspace.name,
     ck.lastupdated = {azure_update_tag},
+    ck.consoleLink = keyspace.consolelink,
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/cosmosdb.py:694]: rename property name

File: cartography/intel/azure/cosmosdb.py [hunk 4]
```diff
@@ -707,6 +720,7 @@ def _load_mongodb_databases(neo4j_session: neo4j.Session, mongodb_databases: Lis
     mdb.location = database.location,
     mdb.region = database.location
     SET mdb.name = database.name,
+    mdb.consoleLink = database.consolelink,
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/cosmosdb.py:723]: same as above

File: cartography/intel/azure/function_app.py [hunk 5]
```diff
@@ -535,48 +539,53 @@ def sync_function_apps_deployments(
...
+    for function in function_apps_list:
+        deployments_list = get_function_apps_deployments_list(
+            function, client, common_job_parameters
+        )
+        function_apps_deployments_list = transform_deployments(deployments_list, function, common_job_parameters)
+        load_function_apps_deployments(
+            neo4j_session,
+            function_apps_deployments_list, update_tag,
+        )
+        cleanup_function_apps_deployments(neo4j_session, common_job_parameters)
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/function_app.py:551]: this function should be outside the for loop

File: cartography/intel/azure/function_app.py [hunk 6]
```diff
@@ -624,50 +634,56 @@ def sync_function_apps_backups(
...
+    for function in function_apps_list:
+        backups_list = get_function_apps_backups_list(
+            function, client, common_job_parameters
+        )
+        function_apps_backups_list = transform_backups(backups_list, function, common_job_parameters)
+        load_function_apps_backups(
+            neo4j_session, function_apps_backups_list,
+            update_tag,
+        )
+        cleanup_function_apps_backups(neo4j_session, common_job_parameters)
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/function_app.py:646]: Take this function outside the for loop

File: cartography/intel/azure/function_app.py [hunk 7]
```diff
@@ -893,14 +922,16 @@ def sync_function_apps_webjobs(
...
+    for function in function_apps_list:
+        webjobs_list = get_function_apps_webjobs_list(
+            function, client,
+        )
+        function_apps_webjobs_list = transform_webjobs(webjobs_list, function, common_job_parameters)
+        load_function_apps_webjobs(
+            neo4j_session, function_apps_webjobs_list,
+            update_tag,
+        )
+        cleanup_function_apps_webjobs(neo4j_session, common_job_parameters)
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/function_app.py:934]: same as above

File: cartography/intel/azure/network.py [hunk 8]
```diff
@@ -768,9 +771,11 @@ def sync_usages(
...
     common_job_parameters: Dict,
 ) -> None:
-    networks_usages_list = get_usages_list(networks_list, client)
-    load_usages(neo4j_session, networks_usages_list, update_tag)
-    cleanup_usages(neo4j_session, common_job_parameters)
+    for network in networks_list:
+        usages = get_usages_list(networks_list, client)
+        networks_usages_list  = transform_usages(usages, network, common_job_parameters)
+        load_usages(neo4j_session, networks_usages_list, update_tag)
+        cleanup_usages(neo4j_session, common_job_parameters)
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/network.py:778]: take this function outside the for loop

**Review by ganeshjagdale9545 — APPROVED** [review]
looks good

**Review by mpurusottamc — APPROVED** [review]
looks good.

**Pull request merged**